### PR TITLE
✨ JS-608: Select by ID

### PR DESCRIPTION
**Pull request by ct-aa**
# Related Tickets
- [JS-608](https://codeandtheory.atlassian.net/browse/JS-608)

# Description
Added the **Select by ID** challenge solutions.

# Validation
* Open the **Select by ID** folder.
* Open the app.js.
* The element with the ID of "btn-phrase" should be assigned to the variable named button.
* The element with the ID of "input-phrase" should be assigned to the variable named input.

# Risks
N/A

### CODE REVIEW

**Review by lycoangelo — CHANGES_REQUESTED** [review]
Hi @ct-aa, the `button` and the `input` doesn't exist on the DOM.

**Comment by ct-aa** [comment]
> Hi @ct-aa, the `button` and the `input` doesn't exist on the DOM.

Thanks, @lycoangelo, This is now updated.

File: Select by ID/index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>Phrase Sayer</title>
+  </head>
+  <body>
+    <p><label for="input-phrase">Type a phrase</label></p>
+    <p><input type="text" id="input-phrase" /></p>
+    <p><button id="btn-phrase">Say Phrase</button></p>
```
Comment (lycoangelo) [inline Select by ID/index.html]: We should not wrap these elements in a `p` tag. This structure violates HTML semantic standards.

**Review by lycoangelo — CHANGES_REQUESTED** [review]

File: Select by ID/index.html (hunk 1 above)
Comment (ct-aa) [reply Select by ID/index.html]: Thanks, @lycoangelo, This is now updated.

**Review by lycoangelo — APPROVED** [review]

**Pull request merged**